app/views/storytime/dashboard/posts/_list.html.erb

```diff
@@ -3,7 +3,12 @@
     <%= link_to [:edit, :dashboard, post], id: "#{post.class.to_s.downcase.split("::").last}_#{post.id}", class: "list-group-item" do %>
       <div class="row">
         <div class="col-md-8">
-           <h4><%= post.title %></h4>
+          <h4><%= post.title %></h4>
+        </div>
+        <div class="col-md-2 col-md-offset-2">
+          <%= form_tag(url_for([:dashboard, post]), method: :delete) do |f| %>
```

Comment:
we should use the same method that we use on other parts of the site to do the delete link. We typically don't use a form tag, just a normal link. I think there's a helper method called `delete_resource_link` to help with that, and there might also be some CSS in the app that will let us show the button on hover without any javascript.